cypress/e2e/page_test.cy.js

```diff
@@ -0,0 +1,74 @@
+const nameInput = () => cy.get('input[name="name"]')
+const specialInput = () => cy.get('input[name="special"]')
+const submitOrder = () => cy.get('button[class="orderBtn"]')
+const pizzaSize = () => cy.get('select[id="size-dropdown"]')
+
+describe('MVP tests', () => {
+  before(() => {
+    cy.visit('http://localhost:3000/pizza')
+  })
+
```

Comment:
You could also add tests to test that clicking the buttons on the home page routes you correctly. (Wasn't needed for MVP but would be something to have in the real world).